Maya/Exporter/BabylonExporter.cs

```diff
@@ -60,13 +60,17 @@ public void Export(ExportParameters exportParameters)
         {
             this.exportParameters = exportParameters;
 
+            //---------------------------------------------------------------------
+            // RD Removed: 07/12/2020: 
+            // The following causes erorrs when trying using mayapy.exe
+            //---------------------------------------------------------------------
             // Check if the animation is running
-            MGlobal.executeCommand("play -q - state", out int isPlayed);
-            if(isPlayed == 1)
-            {
-                RaiseError("Stop the animation before exporting.");
-                return;
-            }
+            // MGlobal.executeCommand("play -q - state", out int isPlayed);
```

Comment:
should this stay comented? if so, then please remove it instead.

Reply:
This command appears to be used to stop playback before exporting begins and should probably be moved to AnimationForm.cs where it can be used as part of the windowed (in gui) export routine.